src/altinn-app-frontend/src/features/instantiate/containers/InstantiateContainer.tsx

```diff
@@ -64,9 +67,16 @@ const InstantiateContainer = () => {
 
     return <UnknownError />;
   }
-
   if (instantiation.instanceId !== null) {
-    return <Redirect to={`/instance/${instantiation.instanceId}`} />;
+    if (!instanceId) {
+      return (
+        <Navigate
+          to={`instance/${instantiation.instanceId}`}
```

Comment:
I don't really know. Think I for the most part moved this file.

Reply:
Hm, you added a new condition check here, so thats a bit worrying I guess. I mean if you added it, but you dont know why? 🙂

Reply:
I realized I had to look at the whole file to recognize what was going on.
In this case `instanceId` is if there is an instance id in the params.
If that is not the case the window should be redirected to a url with the instance id.
Otherwise render the `Outlet`.